### DISCUSSION TITLE
Few questions

### DISCUSSION THREAD

**Discussion by ghost** [vercel/community#1034] [Help] [answered]
1.Can you delete a Vercel account?
2. If you delete a Vercel account, can the phone number be reused in another account?
3. Could I make a REST API with Python in Vercel?
4. What are the limits of the Hobby tier? (By this I mean the limit of projects, etc.)

**Comment by amyegan** [comment, marked as answer]
Hi @tiudvek!

> Can you delete a Vercel account?

Yes, you can delete a Vercel account. To do so, you need to visit [vercel.com/account](https://vercel.com/account) and find the "Delete Personal Account" section at the bottom of the page. But if you already have an account and just want a different username or to change a login connection, you can simply change that in your account settings without all the fuss of deleting and creating a new account.

If you [delete a team](https://vercel.com/guides/how-do-i-delete-a-vercel-team), you can still leave your personal Hobby account open.

> If you delete a Vercel account, can the phone number be reused in another account?

You should be able to use the same phone number for a new account, but unusual activity can sometimes result in a flag that requires manual approval. If you have any trouble creating a new account you can [contact Vercel Support](https://vercel.com/guides/why-can-i-not-signup#contact-vercel-support) for help.

> Could I make a REST API with Python in Vercel?

Depending on your needs and configuration, the [Python Runtime](https://vercel.com/docs/concepts/functions/serverless-functions/runtimes/python#) could allow you to host your API on Vercel. Just be aware of the limitations associated with [Serverless Functions](https://vercel.com/docs/concepts/functions/serverless-functions#limits) and [database connections](https://vercel.com/guides/using-databases-with-vercel).

> What are the limits of the Hobby tier? (By this I mean the limit of projects, etc.)

You can compare the limits for all account types in the Vercel [Limits](https://vercel.com/docs/concepts/limits/overview) documentation, with additional information available in [Usage](https://vercel.com/docs/concepts/limits/usage), [Fair Use Policy](https://vercel.com/docs/concepts/limits/fair-use-policy), and noted throughout the rest of the documentation where relevant.

**Reply by ghost** [reply]
Ok, thank you
